Ball of Duty Server/Domain/Maps/Map.cs

```diff
@@ -186,7 +193,8 @@ private void CheckTimeouts(object sender, ElapsedEventArgs e)
             {
                 Console.WriteLine("removing " + rto);
                 GameObject go;
-                GameObjects.TryRemove(rto, out go);
+                GameObjects.TryGetValue(rto, out go);
+                go.Destroy();
```

Comment:
go might be null here. use TryGetValue correctly.
also we want to remove the go here. not rely on Update to do what we already know needs to be done